code/modules/reagents/reagent_containers/food/snacks/meat.dm

```diff
@@ -448,3 +454,23 @@
 			icon_state = "cozmo_cubes_half"
 		if(70 to INFINITY)
 			icon_state = "cozmo_cubes"
+
+/obj/item/reagent_containers/food/snacks/steak_tartare
+	name = "steak tartare"
+	desc = "Excuse me! Waiter! Is this some kind of joke?! This is raw beef you just SLAPPED onto a plate and sprinkled with condiments! It is COMPLETELY RAW! Even the EGG isn't cooked! Are you kidding me?! Are you trying to give me food poisoning?! What kind of scam are you running here?! No, don't give me that 'it's supposed to be like this' nonsense, the menu said STEAK! Take this back and cook the hell out of it until it stops mooing at me!"
```

Comment:
```suggestion
	desc = "Raw beef sprinkled with condiments. Completely raw, including the egg. Hope it doesn't cause food poisoning."
```
while creative descriptions are fine, this is a bit over the top. i wrote a shorter variant that should keep the spirit of the longer variant, but feel free to write your own.

Reply:
Changing it to something else that's also short. thank you!